### DISCUSSION TITLE
Horse Notes

### DISCUSSION THREAD

**Discussion by Haelis** [Ahrondus/VS_Horse#7] [Ideas]
***check*** Taming/food; Taming; Dry grass = 10%, veggis/grains = 30%, carrots/apples = 60%.
***check*** Breeding; eat and breed like sheep.
***check*** Sheep sounds (place holder). <-- Added horse sounds.
Animations; Bucking, right tail swat
Leashing

**Comment by Haelis** [comment]
Current issues; 

1. Command and mounting take the same key bind, mounting is overriding the commands option.
Key binding mount key to K or another key would probably fix this issue.
_Note; If holding an item in hand can set commands to horse without mounting up._

**Fixed** 2. Terrain stepping; horse is having issues stepping up or down on terrain. Can sprint over most terrain but while walking can't step up one block. 

**Fixed**, can feed with dry grass and carrot bundle (new item). Suggestion to cut said carrot bundle back into regular carrots and give spoil rate.


 **Fixed** Horse doesn't always register when you dismount, hitting Ctrl again after dismounting will prevent the horse from, what looks like, running away from you. Suspected desync, but corrects after hitting ctrl again.

5. Horse comes to a crawl/cannot move in deep water. Water movement needs to be tweaked to be able to get in or out of water.

**Fixed** Horse doesn't drop contents via death.

**Fixed** Needs trough attraction to lure horse.
 
8. Horses hurt their owners sometimes if thrown over a cliff/hill that has them take fall dmg. 

**added**  Horse guide to taming;
"Taming a wild horse is fairly simple.
Right clicking a wild horse with dry grass or carrots starts the taming method. Each feed will increase it's taming % until you complete it and can name it. Feeding it afterwards is advised to raise it's obedience as well so it'll listen better.
Now you can take your new companion on many adventures!
Mounting up on a horse is right click and dismounting is bound to your sneak/crouch key.
Giving a command is right click with any item in hand,
Horses can eat and breed through a feeding trough just as other tamed animals.
Keep an eye out for future progress for saddles, saddlebags and horse attachments!" 

**added** Patch; fruits and grains with the behavior, "consider pet food".

**Comment by Ahrondus** [comment]
Enabled breeding for horses, patched troughs to allow horses to eat from them. Compatible trough items for horses: carrots, spelt grains, rye grains, flax grains, hayblock, dry grass

**Reply by Ahrondus** [reply]
Taming guide is complete

**Reply by Ahrondus** [reply]
Patched fruit and grains to be considered taming foods, and properly are given to horses to tame. Also added sound files.
